clients/launcher/pubspec.yaml

```diff
@@ -1,106 +1,37 @@
 name: launcher
-description: "A launcher for Drivechain"
+description: A new Flutter project.
 publish_to: 'none'
-
-version: 0.0.1
+version: 0.1.0
 
 environment:
-  sdk: '>=3.4.1 <4.0.0'
+  sdk: '>=3.0.3 <4.0.0'
 
 dependencies:
   flutter:
     sdk: flutter
-
-  dart_coin_rpc:
-    git:
-      url: https://github.com/barebitcoin/dart_coin_rpc.git
-      # master, as of 08.06.2024
-      ref: ab0b7cb2de6e1634a6e65892770ed0d157d38443
-  dio: ^5.7.0
-  archive: ^3.4.10
-  path: ^1.8.3
-  path_provider: ^2.1.5
-  stacked: ^3.4.3
-  qr_flutter: ^4.1.0
-  cupertino_icons: ^1.0.8
-  grpc: ^4.0.1
+  auto_route: ^9.3.0+1
+  auto_route_generator: ^9.3.1
```

Comment:
why is this moved from dev_dependencies to dependencies? only used during generation